process-document/src/main/java/com/ritense/processdocument/service/impl/CamundaProcessJsonSchemaDocumentDeploymentService.java

```diff
@@ -84,17 +84,21 @@ public void deployChangedProcessDocumentLinks() throws IOException {
                     continue;
                 }
                 final var documentDefinitionName = filename.split("\\.")[0];
-                final var processDocumentLinkConfigItems = getJson(IOUtils.toString(resource.getInputStream(), UTF_8));
-
-                if (documentDefinitionService.findLatestByName(documentDefinitionName).isPresent()) {
-                    processDocumentLinkConfigItems.forEach(item -> createProcessDocumentLink(documentDefinitionName, item));
-                }
+                final var content = IOUtils.toString(resource.getInputStream(), UTF_8);
+                deploy(documentDefinitionName, content);
             } catch (IOException e) {
                 logger.error("Error while deploying process-document-link", e);
             }
         }
     }
 
+    public void deploy(String documentDefinitionName, String content) throws JsonProcessingException {
```

Comment:
Can you add an integration test for this new method? With the unit test of the importer, we trust on this method to work properly.

Reply:
Added